### PR TITLE
dalvikvm: add Italian translation

### PR DESCRIPTION
**Pull request by iambarge**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.it/android/dalvikvm.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# dalvikvm
+
+> Macchina virtuale Java per Android.
+> Informazioni aggiuntive: <https://source.android.com/devices/tech/dalvik>.
```
Comment (kbdharun) [inline pages.it/android/dalvikvm.md:4]:
```suggestion
> Maggiori informazioni: <https://source.android.com/devices/tech/dalvik>.
```
[Reference](https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py)

File: pages.it/android/dalvikvm.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+- Lancia un programma Java:
+
+`dalvikvm -percorsoclasse {{percorso/al/file.jar}} {{nomeclasse}}`
```
Comment (kbdharun) [inline pages.it/android/dalvikvm.md:8]:
```suggestion
`dalvikvm -classpath {{percorso/al/file.jar}} {{nomeclasse}}`
```
Note: Only translate arguments inside `{{}}` not other parts of command.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**